.github/workflows/notebooks.yml

```diff
@@ -0,0 +1,61 @@
+# Notebook-related checks
+
+name: Notebooks
+
+on:
+  # Relevant PRs
+  pull_request:
+    paths:
+    - "site/en/**"
+  # Allow manual runs
+  workflow_dispatch:
+
+jobs:
+  nbfmt:
+    name: Notebook format
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+    - name: Install tensorflow-docs
+      run: python3 -m pip install -U git+https://github.com/tensorflow/docs
+    - name: Fetch master branch
+      run: git fetch -u origin main:main
+    - name: Check notebook formatting
+      run: |
+        # Only check notebooks modified in this pull request.
+        readarray -t changed_notebooks < <(git diff --name-only main | grep '\.ipynb$' || true)
+        if [[ ${#changed_notebooks[@]} == 0 ]]; then
+          echo "No notebooks modified in this pull request."
+          exit 0
+        else
+          echo "Check formatting with nbfmt:"
+          python3 -m tensorflow_docs.tools.nbfmt --test "${changed_notebooks[@]}"
+        fi
+
+  nblint:
+    name: Notebook lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+    - name: Install tensorflow-docs
+      run: python3 -m pip install -U git+https://github.com/tensorflow/docs
+    - name: Fetch main branch
+      run: git fetch -u origin main:main
+    - name: Lint notebooks
+      run: |
+        # Only check notebooks modified in this pull request.
+        readarray -t changed_notebooks < <(git diff --name-only main | grep '\.ipynb$' || true)
+        if [[ ${#changed_notebooks[@]} == 0 ]]; then
+          echo "No notebooks modified in this pull request."
+          exit 0
+        else
+          echo "Lint check with nblint:"
+          python3 -m tensorflow_docs.tools.nblint \
+            --styles=google,tensorflow \
+            --arg=repo:google/generative-ai-docs --arg=branch:main \
+            --exclude_lint=tensorflow::button_download \
+            --exclude_lint=tensorflow::button_website \
```

Comment:
--exclude_lint=tensorflow::button_website 
 
 I added a button_website, so I think we can drop this if it works.

Reply:
I sent you cl/534605271 to support this. Currently nblint has `tensorflow.org` hard-coded, so I'll leave this here for now and remove once that support is in.